sonar-java-plugin/src/main/java/org/sonar/plugins/java/JavaRulesDefinition.java

```diff
@@ -151,10 +160,16 @@ private static void addMetadata(NewRule rule, @Nullable RuleMetadata metadata) {
     addSecurityStandards(rule, metadata.securityStandards);
   }
 
-  private static void addSecurityStandards(NewRule rule, SecurityStandards securityStandards) {
-    for (String s : securityStandards.OWASP) {
+  private void addSecurityStandards(NewRule rule, SecurityStandards securityStandards) {
+    for (String s : securityStandards.OWASP_2017) {
       rule.addOwaspTop10(OwaspTop10.valueOf(s));
     }
+    boolean isOwaspByVersionSupported = sonarRuntime.getApiVersion().isGreaterThanOrEqual(Version.create(9, 3));
```

Comment:
I feel that this value should be computed only once, at the instantiation of the RulesDefinition class. Currently, it will be called for each rule,